sdk/webpubsub/azure-messaging-webpubsubservice/CHANGELOG.md

```diff
@@ -1,14 +1,12 @@
 # Release History
 
-## 1.0.0b3 (Unreleased)
+## 1.0.0 (2021-11-05)
 
-### Features Added
+### Breaking changes
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- rename operation `generate_client_token` to `get_client_access_token`
+- moved operation `build_authentication_token` into `get_client_access_token` on the client
+- rename parameter `role` to `roles` of operation `get_client_access_token`
```

Comment:
I don't think these are all of the changes, I know we've at least:
1. renamed `expiry_in_minutes`
2. hidden the `operations` folder
3. head calls return booleans instead of raise
4. `has_permission` rename